src/main/java/io/leonis/algieba/Temporal.java

```diff
@@ -0,0 +1,16 @@
+package io.leonis.algieba;
+
+/**
+ * The Interface Temporal.
+ *
+ * This interface describes the functionality of an object which can be timestamped.
+ *
+ * @author Rimon Oz
+ */
+public interface Temporal {
+
+  /**
+   * @return The last seen time of the timestampable object.
```

Comment:
this is actually the value dereference time sometimes (e.g. when the packet doesn't supply an timestamp). I think the javadoc should be improved